lib/generators/react_on_rails/install_generator.rb

```diff
@@ -3,42 +3,44 @@
 module ReactOnRails
   module Generators
     class InstallGenerator < Rails::Generators::Base
+      # fetch USAGE file for details generator description
+      source_root(File.expand_path("../", __FILE__))
+
       # --redux
       class_option :redux,
                    type: :boolean,
                    default: false,
-                   desc: "Install Redux gems and Redux version of Hello World Example",
+                   desc: "Install Redux gems and Redux version of Hello World Example. Default: false",
                    aliases: "-R"
       # --server-rendering
       class_option :server_rendering,
                    type: :boolean,
                    default: false,
-                   desc: "Add necessary files and configurations for server-side rendering",
+                   desc: "Add necessary files and configurations for server-side rendering. Default: false",
                    aliases: "-S"
       # --skip-js-linters
       class_option :skip_js_linters,
                    type: :boolean,
                    default: false,
-                   desc: "Skip installing JavaScript linting files",
+                   desc: "Skip installing JavaScript linting files. Default: false",
                    aliases: "-j"
       # --ruby-linters
       class_option :ruby_linters,
                    type: :boolean,
                    default: false,
-                   desc: "Install ruby linting files, tasks, and configs",
+                   desc: "Install ruby linting files, tasks, and configs. Default: false",
                    aliases: "-L"
       # --ruby-linters
       class_option :heroku_deployment,
                    type: :boolean,
                    default: false,
-                   desc: "Install files necessary for deploying to Heroku",
+                   desc: "Install files necessary for deploying to Heroku. Default: false",
                    aliases: "-H"
-
       # --skip-bootstrap
       class_option :skip_bootstrap,
                    type: :boolean,
                    default: false,
-                   desc: "Skip integrating Bootstrap and don't initialize files and regarding configs",
+                   desc: "Skip integrating Bootstrap and don't initialize files and regarding configs. Default: false",
```

Comment:
@justin808 I feel very strongly that this is adding confusion, not removing it. Now when we say the default is 'false,' is `--skip-bootstrap` false or is `bootstrap` false? This is an established Rails pattern that's been in use for years. 

The only reason you would pass an option is to override the default. Therefore, whatever the option says, that's the opposite of the default. If it says '--redux`, that means the default was not to include redux. If it says `--skip-bootstrap`, that means the default was to include bootstrap. 

I think this repo is already suffering from too much information (TMI). We should be figuring out ways to delete or refactor our documentation to be smaller, not expand it.